.azure-pipelines/documentation_build.yml

```diff
@@ -0,0 +1,38 @@
+jobs:
+  - job: 'documentation'
+    displayName: 'Documentation'
+    pool:
+      vmImage: 'ubuntu-18.04'
+    variables:
+      docs_build_dir: '$(Build.StagingDirectory)/docs_build'
+      data_dir: '$(Build.StagingDirectory)/data'
+    steps:
+      - bash: |
+          git clone https://github.com/scipp/ess-notebooks-data.git $(data_dir)
+          python make_config.py $(data_dir)/ess/v20/imaging
+        displayName: 'Make config'
+      - bash: |
+          set -ex
+          mkdir -p "$(docs_build_dir)"
+        displayName: 'Make directories'
+      - bash: |
+          echo "##vso[task.prependpath]$CONDA/bin"
+        displayName: Add Conda to PATH
+      - bash: |
+          conda env create -f ess-notebook-latest.yml
+        displayName: 'Create Conda environment'
+      - bash: |
+          set -ex
+          source activate ess-notebook-latest
+        displayName: 'Activate Conda environment'
+      - bash: |
+          set -ex
+          source activate ess-notebook-latest
```

Comment:
You activate the conda environment both here and in the previous bash segment.

Reply:
I don't think it persists?

Reply:
Hmm it may well not do. In either case is the `Activate Conda enviroment` bash segment above superfluous?